README.md

```diff
@@ -955,7 +955,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * Free Unmetered Rate Limiting - 1 rule per domain
     * [Cloudflare Pages](https://pages.cloudflare.com/) — Free web hosting (JAMstack platform) for frontend developers to collaborate and deploy websites. 1 build at a time, 500 builds/month, unlimited sites, unlimited requests, unlimited bandwidth.
     * [Cloudflare Workers](https://workers.cloudflare.com/) — Deploy serverless code for free on Cloudflare's global network. 100,000 free requests per day with a workers.dev subdomain.
-    * [Quick Tunnels](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/run-tunnel/trycloudflare) — Create a tunnel from your server to a publically accessible, randomly-generated trycloudflare.com domain. No account required.
+    * [Cloudflare Tunnel](https://www.cloudflare.com/products/tunnel/) - Cloudflare Tunnel can connect HTTP web servers, SSH servers, remote desktops, and other protocols safely to your domains on Cloudflare without a publicly routable IP address. Expose your HTTP resources to the Internet via a public hostname using a DNS record. and use private network routes to expose both HTTP and non-HTTP resources. [Quick Tunnels](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/run-tunnel/trycloudflare) can expose locally running servers over a tunnel to a random subdomain on trycloudflare.com, No account required.
```

Comment:
Generally we post a small one liner about a service, dont need to repro the entire sales brochure here :)

Reply:
yeah. u r right.. too long. 
So we just need to tell reader, use TryCloudflare if u want share a http service.
use Cloudflare Zero Truse Free Plan for more (TCP tunnel, Load balancing , VPN etc..